jobs/aide/spec

```diff
@@ -13,6 +13,9 @@ packages:
 - aide
 
 properties:
-  aide_rules:
-    description: Additional rules for AIDE to apply
+  add_aide_rules:
```

Comment:
I wonder if something like `prepend_rules` and `append_rules` would make more sense?

Reply:
@bengerman13 that's doable and an easy change - give me a few